testData/solutions/win32.json

```diff
@@ -9,16 +9,114 @@
+                    "options\\.Case": "http://registry\\.gpii\\.net/common/case",
+                    "options\\.Filter": "http://registry\\.gpii\\.net/common/filter",
+                    "options\\.MixedCase": "http://registry\\.gpii\\.net/common/mixedCase",
+                    "options\\.Verbosity": "http://registry\\.gpii\\.net/common/verbosity",
+                    "options\\.JAWSPunctuationEnabled": "http://registry\\.gpii\\.net/common/jawsPunctuationEnabled",
+                    "options\\.SayWindowTypeFirst": "http://registry\\.gpii\\.net/common/sayWindowTypeFirst",
+                    "options\\.SayStateFirst": "http://registry\\.gpii\\.net/common/sayStateFirst",
+                    "options\\.InsertKeyMode": "http://registry\\.gpii\\.net/common/insertKeyMode",
+                    "options\\.CaretDetect": "http://registry\\.gpii\\.net/common/caretDetect",
+                    "options\\.CaretDetectTimeOut": "http://registry\\.gpii\\.net/common/caretDetectTimeOut",
+                    "options\\.CaretBlinkRate": "http://registry\\.gpii\\.net/common/caretBlinkRate",
+                    "options\\.SpeakANSIChars": "http://registry\\.gpii\\.net/common/speakANSIChars",
+                    "options\\.IndicateSelected": "http://registry\\.gpii\\.net/common/indicateSelected",
+                    "options\\.SaySelectedFirst": "http://registry\\.gpii\\.net/common/saySelectedFirst",
+                    "options\\.JAWSInsertKey": "http://registry\\.gpii\\.net/common/jawsInsertKey",
+                    "options\\.KeyboardType": "http://registry\\.gpii\\.net/common/keyboardType",
+                    "options\\.UseExtendedKeys": "http://registry\\.gpii\\.net/common/useExtendedKeys",
+                    "options\\.Acronyms": "http://registry\\.gpii\\.net/common/acronyms",
+                    "options\\.Synthesizer": "http://registry\\.gpii\\.net/common/synthesizer",
+                    "options\\.SleepMode": "http://registry\\.gpii\\.net/common/sleepMode",
+                    "options\\.SynthesizerResetFrequency": "http://registry\\.gpii\\.net/common/synthesizerResetFrequency",
+                    "options\\.SimultaneousSynthAndWave": "http://registry\\.gpii\\.net/common/simultaneousSynthAndWave",
+                    "options\\.BrailleDisplay": "http://registry\\.gpii\\.net/common/brailleDisplay",
+                    "options\\.DosScreenReader": "http://registry\\.gpii\\.net/common/dosScreenReader",
+                    "options\\.SayBlankLineCount": "http://registry\\.gpii\\.net/common/sayBlankLineCount",
+                    "options\\.UseWindowsKeyboardHook": "http://registry\\.gpii\\.net/common/useWindowsKeyboardHook",
+                    "options\\.SayCursorShapeChange": "http://registry\\.gpii\\.net/common/sayCursorShapeChange",
+                    "options\\.SayNumericDates": "http://registry\\.gpii\\.net/common/sayNumericDates",
+                    "options\\.SayDollars": "http://registry\\.gpii\\.net/common/sayDollars"
```

Comment:
I know these decisions on what makes a good common term, and value range, etc., are quite difficult, so if you have questions, feel very free to ask for any clarification/guidance necessary. Alternatively, if you want to, we could do a skype call or something to talk through some of them.